Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/HolographicCamera/HolographicCameraNetworkListener.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.HolographicCamera
+{
+    /// <summary>
+    /// Represents the network listener for the camera pose provider app.
+    /// </summary>
+    [RequireComponent(typeof(TCPConnectionManager))]
+    public class HolographicCameraNetworkListener : MonoBehaviour
```

Comment:
>HolographicCameraNetworkListener [](start = 17, length = 32)

This isn't really camera specific, we may just want to call this network listener